### ISSUE TITLE
View of Note Completion Status from History Page

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/best-evidence#122]
@ajburls I'm creating a new issue for [this comment](https://github.com/dwyl/best-evidence/issues/62#issuecomment-318907530) in #62

>on history users WILL be able to see which questions and papers clicked through to have no notes, which have some notes, which have completed notes.

![image](https://user-images.githubusercontent.com/16775804/29267913-a05f0982-80e2-11e7-8c53-70407d4bf156.png)
![completed note](https://user-images.githubusercontent.com/5732209/29275543-65acac6a-8103-11e7-966d-6782013645c2.png)

**Search notes**
- [x] Checkbox appears on new/edit note pages
- [x] Create/amend note symbols vary on the history/recent search pages according to the status

**Publication notes**
- [x] Checkbox appears on new/edit note pages
- [x] Create/amend note symbols vary on the history/recent search pages according to the status

To summarise, users can see which questions and papers have:
1. No notes
2. Some notes
3. Completed notes

Currently (1) has been actioned by Simon's use of the plus icon (this shows there is yet to be a note added). However (2) and (3) are currently displayed by the same icon (the pencil). Is this ok or does a 'completed note' need a different icon to an editable note? Is there a difference between the two? Can a user opt to mark a note as completed?

**Comment by ajburls** [comment]
@markwilliamfirth Completed note needs a different icon  - one for incomplete and one for complete (could be colour (not red/green because of prevalence of this type of colour blindness!) or an outline pencil and a solid pencil etc)  and, yes, user decides if a note is completed. To illustrate with an example, someone might write a note on a paper such as "this looks like a really good resource, must get hold of the full paper". Later they might write "Reading through the whole paper, I see that this is not a randomized controlled trial and provides weak evidence at best" and mark the note as complete as they don't anticipate doing more work or recording more information about the paper.

**Comment by markwilliamfirth** [comment]
@ajburls thanks for confirming that. Can you clarify how a note is marked as complete? Two ways of doing this spring to mind

1. A checkbox that says note is complete above the save note button

2. Two save buttons, "Save as Incomplete" and "Save as Complete"

Please could you let us know which is preferable or if you have a different idea?

**Comment by ajburls** [comment]
@markwilliamfirth I think that option 1 is best with it off as default.

**Comment by markwilliamfirth** [comment]
@SimonLab @Cleop would you be able to time estimate this?

**Comment by ajburls** [comment]
@Cleop This works. Thanks
